src/lazy/encoder/binary/v1_1/flex_sym.rs

```diff
@@ -12,7 +11,6 @@ use crate::RawSymbolRef;
 #[derive(Debug, Clone, Copy)]
 pub enum FlexSymValue<'top> {
```

Comment:
Definitely not something for this PR, just making a note. We can probably convert this to a struct, now that it's just wrapping a symbol ref. Something to come back to later.